var/ramble/repos/builtin/applications/intel-mpi-benchmarks/examples/slurm/execute_experiment.tpl

```diff
@@ -0,0 +1,25 @@
+# This is a template execution script for
+# running the execute pipeline.
+#
+# Variables surrounded by curly braces will be expanded
+# when generating a specific execution script.
+# Some example variables are:
+#   - experiment_run_dir (Will be replaced with the experiment directory)
+#   - command (Will be replaced with the command to run the experiment)
+#   - spack_setup (Will be replaced with the commands needed to setup
+#                  and load a spack environment for a given experiment)
+#   - log_dir (Will be replaced with the logs directory)
+#   - experiment_name (Will be replaced with the name of the experiment)
+#   - workload_run_dir (Will be replaced with the directory of the workload
+#   - application_name (Will be repalced with the name of the application)
+#   - n_nodes (Will be replaced with the required number of nodes)
+#   Any experiment parameters will be available as variables as well.
+
+cd "{experiment_run_dir}"
+
+{spack_setup}
+
+#scontrol show hostnames ${SLURM_JOB_NODELIST} > hostfile
```

Comment:
I'm not sure why this is in a slurm sub-dir if it doesn't use slurm

Reply:
Great catches all round, address in followup commit